.cirrus.yml

```diff
@@ -103,16 +103,19 @@ build_task:
               CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
               # ID for re-use of build output
               CI_DESIRED_RUNTIME: crun
+              CI_DESIRED_NETWORK: netavark
         - env: &priorfedora_envvars
               DISTRO_NV: ${PRIOR_FEDORA_NAME}
               VM_IMAGE_NAME: ${PRIOR_FEDORA_CACHE_IMAGE_NAME}
               CTR_FQIN: ${PRIOR_FEDORA_CONTAINER_FQIN}
               CI_DESIRED_RUNTIME: crun
+              CI_DESIRED_NETWORK: netavark
```

Comment:
I thought prior fedora should use cni?

Reply:
Ugh - that's what happens when I code late in the day. Thank you for catching that. Fixed and pushed.